Modules/Broker/BrokerModule.lua

```diff
@@ -0,0 +1,40 @@
+BattlePetCompletionist = LibStub("AceAddon-3.0"):GetAddon("BattlePetCompletionist")
```

Comment:
Please add the license header above as well

Reply:
Added.